tests/tensor_search/test_add_documents_use_existing_tensors.py

```diff
@@ -534,4 +539,4 @@ def test_use_existing_tensors_all_data_types(self):
                 config=self.config, index_name=self.index_name_1,
                 document_ids=[doc["_id" ]for doc in doc_arg], show_vectors=True)
             
-            self.assertEqual(d1, d2)
\ No newline at end of file
+            self.assertAlmostEqual(d1, d2)
```

Comment:
i think you are comparing the vectors here. do you want to use np.allclose and set a threshold?

Reply:
Looking at other tests, I decided to change back to `assertEqual`, as I don't think that was the cause of the error anyway, it was the different data types being put in the same field.

We have had no problems directly comparing doc vector embeddings in other tests, as seen here:
test_non_tensor_field_list: https://github.com/marqo-ai/marqo/blob/mainline/tests/tensor_search/test_add_documents.py#L1127
test_use_existing_tensors_dupe_ids: https://github.com/marqo-ai/marqo/blob/mainline/tests/tensor_search/test_add_documents_use_existing_tensors.py#L143

Reply:
thanks for the in-depth description. approved